### ISSUE TITLE
Online Music Database

### ISSUE THREAD

**Issue by therealdvl** [therealdvl/musicdb#1]
Immediate Plans
1) Database tracking artist careers, discographies and collaborations with other artists.
2)Aggregation of album and live show reviews (both professional reviews and user created reviews)
3) Wiki of artist information (history/influences/similar artists, etc.) User created & reviewed

Eventual/Possibilities
Tour News
Venue News
Record Label Database
Lyrics Database
Genre Database

Long term goals:
Platform for crowd funded artist booking

Influences:
IMDB
Rotten Tomatoes
discogs.com
quantonemusic.com
allmusic.com

**Comment by therealdvl** [comment]
seeing that this project would be massive, it would be smart to start this off as a database of local musicians and expand as time allows

**Comment by jordnfk** [comment]
agreed. I'm not as fluent with music as you are so I may not be as much help on that side of things.
